client-java/src/main/java/uk/gov/gchq/palisade/client/internal/impl/Configuration.java

```diff
@@ -99,7 +94,8 @@
     private static final String KEY_SERVICE_FRS_PATH = "service.filteredResource.path";
 
     /**
-     * The path portion of the Data service URI which defaults to "data/read/chunked"
+     * The path portion of the Data Service URI which defaults to
+     * "data/read/chunked"
```

Comment:
is this still "data/read/chunked" @dev930018 added a comment about migrating to just read/chunked

Reply:
It should be `read/chunked/`. Which is still wrong as the path should reflect the type of resource and not how it is accessed.

Reply:
fixed